sdks/java/io/hadoop-input-format/src/test/resources/kubernetes/cassandra/data-load.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Identify the pod
+cassandra_pods="kubectl get pods -l name=cassandra1"
+running_seed=$($cassandra_pods --no-headers | \
```

Comment:
As per our discussion, fetched kubectl status in json format. Below command is getting executed successfully:
kubectl get svc cassandra1 -o jsonpath
We have removed use of awk from the script and used this command instead.